documentation/projects/proposals/rekognition_data/project_proposal.md

```diff
@@ -0,0 +1,142 @@
+# 2024-03-20 Project Proposal: Incorporate Rekognition data into the Catalog
+
+**Author**: @AetherUnbound
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @stacimc
+- [ ] @obulat
+
+## Project summary
+
+<!-- A brief one or two sentence summary of the project's features -->
+
+[AWS Rekognition data][aws_rekognition] in the form of object labels was
+collected by Creative Commons several years ago for roughly 100m image records
+in the Openverse catalog. This project intends augment the existing tags for the
```

Comment:
Would be nice to add the link to CC blog post on the AWS grant: https://creativecommons.org/2019/12/05/cc-receives-aws-grant-to-improve-cc-search/